include/torch-mlir/Dialect/Torch/IR/GeneratedAtenOps.td

```diff
@@ -1297,6 +1297,23 @@ def Torch_AtenThresholdBackwardOp : Torch_Op<"aten.threshold_backward", [
   let assemblyFormat = "$grad_output `,` $self `,` $threshold attr-dict `:` qualified(type($grad_output)) `,` qualified(type($self)) `,` qualified(type($threshold)) `->` qualified(type($result))";
 }
 
+def Torch_AtenUniform_Op : Torch_Op<"aten.uniform_", [
+    IsTrailingUnderscoreInplaceVariant,
```

Comment:
I don't think this should have IsTrailingUnderscoreInplaceVariant, because that will cause ReduceOpVariants to convert it to `aten.uniform` which doesn't exist.

Reply:
Oh, I see that you updated that pattern to handle this specially.

Reply:
I asked @bdhirsh about this in a chat.

Reply:
oh, I forgot to run `./build_tools/update_torch_ods.sh`. This trait should be removed.